cdap-ui/app/cdap/components/SortableTable/index.js

```diff
@@ -27,13 +27,15 @@ export default class SortableTable extends Component {
     this.state = {
       entities: props.entities,
       sortByHeader: '',
-      sortOrder: 'asc'
+      sortOrder: 'asc',
+      sortOnInitialLoad: typeof this.props.sortOnInitialLoad !== 'boolean' ? true : this.props.sortOnInitialLoad
     };
-
-    this.renderTableBody = this.props.renderTableBody.bind(this);
   }
 
   componentWillMount() {
+    if (!this.props.sortOnInitialLoad) {
```

Comment:
Just to make sure, should this be `this.props.sortOnInitialLoad` or `this.state.sortOnInitialLoad`? 'cause you haven't used the state one so far yet in this component.